service/app/controllers/coursemanagement/validator/CourseCreateRequestValidator.java

```diff
@@ -0,0 +1,49 @@
+package controllers.coursemanagement.validator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.text.MessageFormat;
+import java.util.Map;
+
+import static org.sunbird.common.responsecode.ResponseCode.SERVER_ERROR;
+
+public class CourseCreateRequestValidator {
+    public static void validateRequest(Request request) {
+        try {
+            String message = "";
+            if (null == request.get("course")) {
+                message += "Please provide course";
```

Comment:
CREATE: {
    MISSING_CODE: 'ERR_COURSE_CREATE_FIELDS_MISSING',
    MISSING_MESSAGE: 'Required fields for create course are missing'
  },

Log using message: `'Error due to missing request body or course '`